src/apps/frontend/types/auth.ts

```diff
@@ -3,10 +3,22 @@ import { JsonObject } from './common-types';
 export class AccessToken {
   accountId: string;
   token: string;
+  expiresAt:Date;
 
   constructor(json: JsonObject) {
     this.accountId = json.account_id as string;
     this.token = json.token as string;
+    // Convert from Unix timestamp (seconds) to Date object
+    this.expiresAt = new Date(parseFloat(json.expires_at as string) * 1000);
+  }
+  
+  toJson(): JsonObject { 
+    return{ 
+      account_id:this.accountId,
+      // Convert Date object back to Unix timestamp (seconds)
```

Comment:
Avoid comments like this. See: [How to write better code comments](https://dev.to/adammc331/todo-write-a-better-comment-4c8c)